Tests/test_PDB.py

```diff
@@ -995,6 +995,78 @@ def test_get_atoms(self):
         self.assertEqual(len(atoms), 756)
 
 
+class ChangingIdTests(unittest.TestCase):
+
+    def setUp(self):
+        with warnings.catch_warnings():
+            warnings.simplefilter("ignore", PDBConstructionWarning)
+            self.struc = PDBParser(PERMISSIVE=True).get_structure(
+                                                  'X', "PDB/a_structure.pdb")
+
+    def test_change_model_id(self):
+        """Change the id of a model"""
+        for model in self.struc:
+            break  # Get first model in structure
```

Comment:
Assuming this works it would be more elegant (here and the other examples below):

```
model = next(self.struc)
```

This will also give a clear error if there were ever no models.